scripts/ci

```diff
@@ -8,4 +8,5 @@ cd $(dirname $0)
 ./validate
 ./validate-ci
 ./package
+./package-for-ci
```

Comment:
These two scripts are doing duplicate work it seems and I'm not sure why that's necessary.